resources/js/Components/MismatchesTable.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <wikit-table>
+  <TableComponent>
```

Comment:
I think we use `kebab-case` for component tags, instead of `UpperCamelCase`, we can continue to do so for the sake of consistency. What do you think?

Reply:
I agree. I didn't know it was optional. :confused:

Reply:
Done :+1:  We could consider a linting rule for this if we don't want it to be optional.

Reply:
We should have a day-long meeting to decide our linting rules I think :D